src/Helper/Fields/Field_Products.php

```diff
@@ -318,13 +318,15 @@ public function value() {
 
 				/* Store product in $form_array array */
 				$form_array['products'][ $id ] = [
```

Comment:
@jestonihpi revert this formatting change. It isn't relevant to your PR.

Reply:
I have added these data on form_array Jake for the quantity display changes.